File: C++/Begineer Problem/Logic Building (Patterns)/pattern13.cpp
#include<bits/stdc++.h>
using namespace std;

void pattern13(int n) {
    int count = 1;
    for(int i = 1; i <= n; i++){
        for(int j = 1; j <= i; j++){
            cout << count << " ";
            count++;
        }
        cout << endl;
    }
}
int main()
{
    int testCases;
    cin >> testCases;
    while (testCases--){
        int n;
        cin >> n;
        pattern13(n);
    }
    return 0;
}

File: Java/Logic Building (Patterns)/pattern13.java
import java.util.Scanner;

public class pattern13 {
    public static void pattern13(int n) {
        int count = 1;
        for(int i = 1; i <= n; i++){
            for(int j = 1; j <= i; j++){
                System.out.print(count + " ");
                count++;
            }
            System.out.println();
        }
    }
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int testCases = sc.nextInt();
        while(testCases-- > 0){
            int n = sc.nextInt();
            pattern13(n);
        }
    }
}
